src/pages/guides/theming/index.md

```diff
@@ -0,0 +1,397 @@
+- **plugins** - The `plugins` property configures Tailwind CSS plugins. See [Plugins](https://v2.tailwindcss.com/docs/plugins) from the Tailwind documentation.
+- **separator** - The `separator` property configures the character used to separate the variant prefixes from the utility names. Tailwind uses the colon `:` by default, but PWA Studio uses the underscore (`_`) because CSS Modules doesn't support the colon in class names. See [Separator](https://v2.tailwindcss.com/docs/configuration#separator) from the Tailwind documentation.
+- **theme** - The `theme` property is for your custom values used to override or extend the Tailwind CSS framework. See [Theme Configuration](https://v2.tailwindcss.com/docs/theme) from the Tailwind documentation.
+- **matcher** - The `matcher` variable is used to match CSS Modules classnames that use the `composes` function. This is used by PurgeCSS to remove unused CSS from the `index.css` file. See [Writing purgeable HTML](https://v2.tailwindcss.com/docs/optimizing-for-production#writing-purgeable-html) from the Tailwind documentation.
+
+<InlineAlert slots="text" />
+
+When looking at code samples in the Tailwind docs, keep in mind that the base project configures Tailwind to use `_` as the [seperator](https://v2.tailwindcss.com/docs/configuration#separator) instead of the default colon (`:`).
+
+### Add postcss.config.js
+
+The `postcss.config.js` file is a standard configuration file used by the `postcss-loader` package. So you need to add it to your project's root and configure it to add `tailwindcss` and the `autoprefixer` as a PostCSS plugins for your app. Your `postcss.config.js` file should look like this:
+
+The `create-pwa` CLI adds and configures this file as follows:
+
+```js
+// postcss.config.js
+
+module.exports = {
+  plugins: [
+    require('autoprefixer'),
+    require('tailwindcss')('./tailwind.config.js'),
+  ],
+};
+```
+
+## Step 3: Test your configuration
+
+To test your configuration with the `[venia]` theme set as a `preset` in your config file, add the following `theme` style override and rebuild your app.
+
+```js
+// tailwind.config.js
+
+theme: {
+  extend: {
+    textColor: {
+      colorDefault: '#BB0000'
+    }
+  }
+}
+```
+
+If the text color in your Venia-themed app turns red, your configuration is working correctly. To test your own theme, refer to [Customizing your theme](https://tailwindcss.com/docs/adding-custom-styles#customizing-your-theme) from the Tailwind documentation.
+
+## Step 4: Customize the base styles
+
+You have two options for customizing the Tailwind and Venia base styles:
+
+1. Update the `tailwind.config.js` file directly.
+1. Import your own theme file as a `preset` (recommended).
+
+### Option 1: Update tailwind.config.js theme directly
+
+If you followed Step 3 above, you have already done this as a test. Another example: change the default font family by adding new styles to the `tailwind.config.js` `theme` property:
```

Comment:
I would assume that the reader followed step 3 (no need to write "above"). Add the fontFamily example below to step 3, and use this section point out this is what you did, but you probably don't want to use it in a real-world scenario.